### ISSUE TITLE
Doesn’t support search for Unicode (eg Chinese)

### ISSUE THREAD

**Issue by realliyifei** [codycodes/alfred-books#11]
My books library is bilingual, ie English and Chinese, but the search function doesn't support the Chinese search (and probably other non-alphameric languages). 

I provide a Chinese epub book [未来简史](https://b-ok.global/book/3639761/7c40e4?dsource=recommend) (A Brief History of Tomorrow) here in case you need it for testing.

**Comment by codycodes** [comment]
Thanks for letting me know! I tested with the book you linked and it's working for me; feel free to lmk if it's still not working for you. I pushed a new release: #12 

You can update the workflow from within Alfred manually (if using the default keyword `ib`) using:

```
ib workflow:update
```

**Comment by realliyifei** [comment]
Thanks for fixing it! But it only works for one character.

One char: WORKS

<img width="556" alt="image" src="https://user-images.githubusercontent.com/32441766/92990389-b6984080-f50e-11ea-83fb-eb93b55c4323.png">

More than two chars: NOTHING

<img width="560" alt="image" src="https://user-images.githubusercontent.com/32441766/92990396-c9ab1080-f50e-11ea-8bf3-58be596087a7.png">

**Comment by codycodes** [comment]
Oof, that's a bit embarrassing. Sorry about that! I should probably have some tests for this. I'll work on a fix but it may be a couple of days so I can ensure it's thorough. Thank you for giving feedback on this workflow!

**Comment by codycodes** [comment]
TL;DR: latest release fixes the issue: https://github.com/codycodes/alfred-books/releases/tag/v1.0.5

You can update with the following manually:

```
ib workflow:update
```

I've attempted to test what caused the workflow to fail on multiple unicode chars and found what failed, but am not sure what exactly is happening. After doing some reading up on unicode decoding/encoding, I learned more about what should be happening, but I haven't been able to test it to make sure. When I wrote this workflow originally I was following examples I found but realize that when I was calling `' '.join(book.title)` python was probably silently promoting strings to ASCII even though I have now specified unicode strings as default. At any rate I simply removed those string methods and have tested all the filtering methods on a couple different books. It seems to be working well. It would be great to find some sort of testing framework to help, but I do believe I resolved the issue.

Thanks again for reporting this issue and contributing ideas! Please feel free to reopen this issue if you're still facing issues.

**Comment by realliyifei** [comment]
Thank you. It works BEAUTIFULLY!
